eox_core/middleware.py

```diff
@@ -36,13 +36,28 @@ class PathRedirectionMiddleware(object):
 
     def process_request(self, request):
         """
-        This middleware processes the path of every request and determines if there
-        is a configured action to take.
+        Process the path of every request and
+        determine the correct custom redirect.
         """
 
-        if not configuration_helper.has_override_value("EDNX_CUSTOM_PATH_REDIRECTS"):
-            return None
+        custom_configs = {
```

Comment:
I would name this dict as custom_redirection_settings

Reply:
Done.